interfaces/builtin/autopilot_introspection.go

```diff
@@ -0,0 +1,108 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const introspectionConnectedPlugAppArmor = `
+# Description: Can introspect an application with a testability library.
+# Restricted because this gives privileged access to the application which
+# enabled testability
+# Usage: reserved
+
+dbus (send, receive)
+    bus=session
+    path=/com/canonical/Autopilot/**
+    interface=org.freedesktop.DBus.Introspectable
+    member=Introspect
+    peer=(label=unconfined),
+
+dbus (send, receive)
+    bus=session
+    path=/
+    interface=org.freedesktop.DBus.Introspectable
+    member=Introspect
+    peer=(label=unconfined),
+
+dbus (send, receive)
+    bus=session
+    path=/com
+    interface=org.freedesktop.DBus.Introspectable
+    member=Introspect
+    peer=(label=unconfined),
+
+dbus (send, receive)
+    bus=session
+    path=/com/canonical
+    interface=org.freedesktop.DBus.Introspectable
+    member=Introspect
+    peer=(label=unconfined),
+
+dbus (send, receive)
+    bus=session
+    path=/com/canonical/Autopilot
+    interface=org.freedesktop.DBus.Introspectable
+    member=Introspect
+    peer=(label=unconfined),
+
+dbus (send, receive)
+    bus=session
+    path=/com/canonical/Autopilot/Introspection
+    interface=com.canonical.Autopilot.Introspection
+    member=GetVersion
+    peer=(label=unconfined),
+
+dbus (send, receive)
+    bus=session
+    path=/com/canonical/Autopilot/Introspection
+    interface=com.canonical.Autopilot.Introspection
+    member=GetState
+    peer=(label=unconfined),
```

Comment:
In each those cases the peer is unconfined so it must come from classic. If that's the design you want then I'd suggest adjusting the documentation and documenting this in `docs/interfaces.md`